examples/window/screenshot.rs

```diff
@@ -1,9 +1,10 @@
 //! An example showing how to save screenshots to disk
 
-use bevy::prelude::*;
-use bevy::window::SystemCursorIcon;
-use bevy_render::view::cursor::CursorIcon;
-use bevy_render::view::screenshot::{save_to_disk, Capturing, Screenshot};
+use bevy::{prelude::*, window::SystemCursorIcon};
+use bevy_render::view::{
+    cursor::CursorIcon,
+    screenshot::{save_to_disk, Capturing, Screenshot},
+};
```

Comment:
probably for another PR: this example shouldn't use bevy_render directly. we used to have a CI that errored when that happens

Reply:
Addressed in #15333